Source/Core/VideoBackends/OGL/TextureCache.h

```diff
@@ -8,54 +8,45 @@
 
 #include "Common/CommonTypes.h"
 #include "Common/GL/GLUtil.h"
+#include "VideoBackends/OGL/ProgramShaderCache.h"
 
 #include "VideoCommon/TextureCacheBase.h"
 #include "VideoCommon/VideoCommon.h"
 
+class AbstractTexture;
+struct TextureConfig;
+
 namespace OGL
 {
+
+// TODO: Don't do this
```

Comment:
@phire and @stenzek and I talked about this, honestly there's not really a clean way to pass this around atm.  They have a grand vision..this is a simple stepping stone for them.